FrontEnd/src/components/basics/ImageUpload/ImageUpload.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { FunctionComponent, ChangeEvent, useState } from 'react';
+import { Row } from 'react-bootstrap';
+import { FilledImage } from '@basics';
+import { imageUpload } from '@icons';
+import { Button } from '@basics';
+import styles from './ImageUpload.module.scss';
+import cn from 'classnames';
+
+interface UploadButtonProps {
+  handleChange: (e: ChangeEvent<HTMLInputElement>) => void;
+}
+
+const UploadButton: FunctionComponent<UploadButtonProps> = ({
+  handleChange,
+}) => (
+  <div className={styles.image_input_wrapper}>
+    <label htmlFor="image_upload" className={styles.image_input_label}>
+      <imageUpload.Add />
+    </label>
+    <input
+      id="image_upload"
+      type="file"
+      onChange={handleChange}
+      multiple
+      accept="image/*"
+      className={styles.image_input}
+    />
+  </div>
+);
+
+interface IndividualImageProps {
+  index: number;
+  file: File;
+  onDelete: (currentIndex: string) => void;
+}
+
+const IndividualImage: FunctionComponent<IndividualImageProps> = ({
+  index,
+  file,
+  onDelete,
+}) => {
+  const fileUrl = URL.createObjectURL(file);
+  return (
+    <div className={styles.single_image}>
+      <div className={styles.delete_wrapper_background}>
+        <Button
+          variant="wrapper"
+          className={styles.delete_wrapper}
+          onClick={() => onDelete(index.toString())}
+        >
+          <imageUpload.Delete className={styles.delete} />
+        </Button>
+      </div>
+      <FilledImage
+        src={fileUrl}
+        alt={'uploaded images'}
+        className={styles.round_image}
+      />
+    </div>
+  );
+};
+
+const ImageUpload: FunctionComponent = () => {
```

Comment:
Should probably add component header to explain to others how to use this component

Reply:
bad code. I rewrote it. No need for comments